### DISCUSSION TITLE
Can you bypass the 25w SPL limit on the Rog Ally?

### DISCUSSION THREAD

**Discussion by inzork** [seerge/g-helper#3131] [Q&A] [answered]
Currently I'm rocking the Ally + [this](https://www.alza.hu/EN/vention-type-c-usb-c-to-hdmi-3x-usb30-pd-docking-station-015m-grey-mirrored-surface-type-d6253190.htm) usb-c hub (power delivery 3.0, max 87w) but when I plug in the the charger that came with the Ally I can only set the SPL to 25w in Armoury Crate. 

If I set the SPL, SPPT and FPPT to 30w in g-helper and apply the power limits, will that bypass the 25w limit set by asus because I have an "insufficient charger"?

ps: I don't really understand why the limit is there in the first place when the stock charger is 65w.

**Comment by seerge** [comment, marked as answer]
@inzork hello, most likely you can :) At least on my Ally (non X) it runs 30W just fine even from the battery. 

But please keep in mind that it's not much benefits from that in a first place due to diminishing returns on AMD APU here.

**Reply by inzork** [reply]
That's great news, thank you! I use the Ally as a desktop replacement and League of Legends runs poorly on 25w :(.

Oh and one more thing, do you know by any chance what's the RPM equivalent of armoury crate 100% fan speed? I'd like to use the same fan curve I have there (tried calibrate, but it sounded like the fans reached the max speed long before the program stopped displaying higher and higher numbers).

Thank you again! :)

**Reply by seerge** [reply]
BIOS interprets those % in some strange way. 1% is usually something like 1600RPM, while 100% is 5600RPM+ (depends on device).
You can just switch Y-axis in fan editor to % **by clicking on it** to just mirror what you had in AC :)

**Reply by inzork** [reply]
Can't thank you enough for the help! Even 35w on all 3 sliders works without the battery draining, and the game runs waaay better.

**Reply by seerge** [reply]
Glad to hear
